visidata/canvas.py

```diff
@@ -808,6 +865,13 @@ def deleteSourceRows(self, rows):
 Canvas.addCommand('g'+ENTER, 'dive-visible', 'vs=copy(source); vs.rows=list(rowsWithin(plotterVisibleBox)); vd.push(vs)', 'open sheet of source rows visible on screen')
 Canvas.addCommand('gd', 'delete-visible', 'deleteSourceRows(rowsWithin(plotterVisibleBox))', 'delete rows on source sheet visible on screen')
 
+Canvas.addCommand('', 'add-line-x', 'sheet.gridlines_x += [float(x) for x in input("add line(s) at x = ", value="0.0", defaultLast=True).split()]; sheet.refresh()', 'draw a vertical line at x-values (space-separated)')
+Canvas.addCommand('', 'add-line-y', 'sheet.gridlines_y += [float(y) for y in input("add line(s) at y = ", value="0.0", defaultLast=True).split()]; sheet.refresh()', 'draw a horizontal line at y-values (space-separated)')
```

Comment:
These should use `col.type()` (dates are a numeric type).  Ideally the default value should be something reasonable, maybe the median of the data range?  And let's use `type="gridlinex"`/gridliney with both the add and remove, so people can scroll up to remove the most recently added gridline.

For the longnames, let's not use the 'add/remove/clear' verbs since current 'add-' commands are used to modify the data.  What can we use to indicate this is an interface-only command?

Reply:
Instead of 'add/remove/clear', how about 'draw/erase/erase'?

Reply:
That works for me!  `draw-` especially is very clearly interface only.